skunkbooth/filters/card_deck.py

```diff
@@ -0,0 +1,33 @@
+from random import randint
+from typing import List, Tuple
+import random
+
+
+class filter:
+    """Sample filter for ASCII operations"""
```

Comment:
Maybe update all of these to be the name of the filter

Reply:
Done!